sparse_strips/vello_toy/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "vello_toy"
+version.workspace = true
+description = "A collection of different binary crates for interacting with the sparse rendering stack."
+categories = ["rendering", "graphics"]
+keywords = ["2d", "vector-graphics"]
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+repository.workspace = true
+# Prevent accidental publishing until the initial release
```

Comment:
```suggestion
```
As in https://github.com/linebender/vello/blob/a61783396b6d00f187235b2b1fccb0d91144d92a/vello_tests/Cargo.toml#L7-L9